aporia.nim

```diff
@@ -125,11 +125,46 @@ proc updateTabUI(t: Tab) =
   else:
     t.label.setText(name)
   setTabTooltip(t)
+ 
+proc checkFileUpdate(pageNum : int): int = 
+  ## Checks if the current tab open has been updated outside of Aporia.
+  ## Returns ``1`` if the file has been updated, ``0`` if not, and ``-1``
```

Comment:
Oh. I see. You ask the user whether they want to reopen it in the same proc. Please separate this logic.